formfyxer/lit_explorer.py

```diff
@@ -972,6 +972,39 @@ def get_citations(text: str, tokenized_sentences: List[str]) -> List[str]:
     return citations_with_context
 
 
+def get_sensitive_fields(fields: List[str]) -> List[str]:
+    """
+    Given a list of fields, identify those related to sensitive information. Sensitive fields include 
```

Comment:
```suggestion
    Given a list of fields, identify if any are related to sensitive information. Sensitive fields include 
```

Unless you change it go map to individual names.